v3/internal/setupwizard/notarize.go

```
@@ -0,0 +1,260 @@
+
+	args := notarizeArgs(profileName, appleID, teamID)
+	job := &notarizeJob{
+		command:     shellJoin(args),
+		dir:         dir,
+		profileName: profileName,
+		teamID:      teamID,
+		state:       notarizeStateRunning,
+	}
+
+	if err := os.WriteFile(job.scriptPath(), []byte(notarizeScript(args, job.statusPath())), 0o700); err != nil {
+		os.RemoveAll(dir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)